tests/kernel/timer/timer_api/src/main.c

```diff
@@ -45,8 +45,8 @@ struct timer_data {
  * between the two clocks.  Produce a maximum error for a given
  * duration in microseconds.
  */
-#define BUSY_SLEW_THRESHOLD_TICKS(_us)				\
-	k_us_to_ticks_ceil32((_us) * BUSY_TICK_SLEW_PPM		\
+#define BUSY_SLEW_THRESHOLD_TICKS(_us)					\
+	k_us_to_ticks_ceil32((_us) * (uint64_t)BUSY_TICK_SLEW_PPM	\
```

Comment:
... but clearly the overflow fix here is obviously correct.  No objections to this patch.